extensions/CreateBox/CreateBox.php

```diff
@@ -132,8 +132,11 @@ function acRedirect( $title, $action ) {
 	global $wgRequest, $wgOut, $wgUser;
 
 	$actionParam = 'action';
+	$params = $wgRequest->getValues();
+	$preload = $params['preload'];
+	$preloadIsMediaWikiPage = !empty( $preload ) && strpos($preload, 'MediaWiki:') === 0;
```

Comment:
I recommend to check with Title:isInNamespace instead of strpos due to possibility of encountering language-specific namespace aliases - even though for NS_MEDIAWIKI it is not a common thing, you can find some examples in the languages dir of the application repo.